lib/bpk-component-chip/src/BpkChip.js

```diff
@@ -72,53 +79,74 @@ const dynamicStyles = BpkDynamicStyleSheet.create({
     }),
   },
   innerSelected: {
-    backgroundColor: colorSkyBlue,
+    backgroundColor: {
+      light: privateChipDefaultOnBackgroundDay,
+      dark: privateChipDefaultOnBackgroundNight,
+    },
   },
   innerDisabled: {
-    backgroundColor: backgroundTertiaryColor,
+    backgroundColor: {
+      light: privateChipDisabledBackgroundDay,
+      dark: privateChipDisabledBackgroundNight,
+    },
   },
   text: {
     color: textPrimaryColor,
     includeFontPadding: false,
   },
   textSelected: {
-    color: colorWhite,
+    color: { light: textOnDarkDay, dark: textOnDarkNight },
   },
   textDisabled: {
     color: disabledTextColor,
   },
   icon: {
-    color: { light: colorSkyGrayTint02, dark: colorSkyGrayTint05 },
+    color: textPrimaryColor,
     includeFontPadding: false,
     marginStart: spacingSm,
   },
+  iconSelected: {
+    color: { light: textOnDarkDay, dark: textOnDarkNight },
+  },
   iconDisabled: {
     color: disabledTextColor,
   },
   innerOutline: {
-    backgroundColor: 'transparent',
-    borderColor: lineDarkColor,
-    borderWidth: borderSizeSm,
+    backgroundColor: {
+      light: privateChipOnDarkNormalBackgroundDay,
+      dark: privateChipDefaultNormalBackgroundNight,
+    },
     paddingVertical: spacingMd - borderSizeSm,
   },
   innerOutlineSelected: {
-    backgroundColor: colorSkyBlue,
+    backgroundColor: {
+      light: privateChipOnDarkOnBackgroundDay,
+      dark: privateChipDefaultOnBackgroundNight,
```

Comment:
same here - `privateChipOnDarkOnBackgroundNight`?

Reply:
In dark mode in Figma the onDark is the same as the default in dark mode